sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -468,6 +468,16 @@ public int update(@NonNull String table, @NonNull ContentValues values,
      */
     @WorkerThread
     boolean yieldIfContendedSafely(long sleepAmount, TimeUnit sleepUnit);
+
+    /**
+     * Closes the object and release any system resources it holds.
```

Comment:
Can we make this whole thing say, `Equivalent to calling {@link #end()}.`

And then in the implementation switch `close()` to call `end()` instead of `end()` calling `close()`.

Also, change the test so it no longer throws `IOException`?